clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy

```diff
@@ -17,9 +17,12 @@
 package com.netflix.spinnaker.clouddriver.aws.cache
 
 import com.netflix.frigga.Names
-import static com.netflix.spinnaker.clouddriver.aws.AmazonCloudProvider.AWS
+import com.netflix.spinnaker.clouddriver.aws.AmazonCloudProvider
 
 class Keys {
+
+  static def AWS = AmazonCloudProvider.AWS
```

Comment:
I'd be more in favor of the AmazonCloudProvider change to make that constant public